CharacterCreator/CharacterCreator/Models/Characters.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CharacterCreator.Models
+{
+    public enum Sex
```

Comment:
Put in separate file called Sex.cs

I would recommend making a folder inside of Models called Enum and placing your enums there

Reply:
Fixed